autoscheduler/frontend/src/tests/redux/CourseCards.test.ts

```diff
@@ -593,4 +595,279 @@ describe('Course Cards Redux', () => {
+        instructor: new Instructor({
+          name: 'Test',
+        }),
+        honors: false,
+        remote: true,
+        asynchronous: false,
+        grades: null,
+      };
+      const dummyMeetingArgs: any = {
+        id: 1,
+        building: 'BILD',
+        meetingDays: new Array(7).fill(true),
+        startTimeHours: 0,
+        startTimeMinutes: 0,
+        endTimeHours: 0,
+        endTimeMinutes: 0,
+        meetingType: MeetingType.LEC,
+        section: null,
+      };
+      const testSections: Section[] = [
+        new Section({
+          ...dummySectionArgs,
+          sectionNum: '501',
+          grades: {
+            gpa: 3.0,
+          },
+          instructor: new Instructor({ name: 'Alice' }),
+          currentEnrollment: 11,
+          honors: true,
+        }),
+        new Section({
+          ...dummySectionArgs,
+          sectionNum: '502',
+          grades: {
+            gpa: 2.7,
+          },
+          instructor: new Instructor({ name: 'Zander' }),
+          currentEnrollment: 49,
+          honors: false,
+        }),
+        new Section({
+          ...dummySectionArgs,
+          sectionNum: '503',
+          grades: {
+            gpa: 3.6,
+          },
+          instructor: new Instructor({ name: 'Tyagi' }),
+          currentEnrollment: 7,
+          honors: false,
+        }),
+        new Section({
+          ...dummySectionArgs,
+          sectionNum: '504',
+          grades: {
+            gpa: 3.7,
+          },
+          instructor: new Instructor({ name: 'John' }),
+          currentEnrollment: 31,
+          honors: false,
+        }),
+        new Section({
+          ...dummySectionArgs,
+          sectionNum: '505',
+          grades: {
+            gpa: 3.4,
+          },
+          instructor: new Instructor({ name: 'Tyagi' }),
+          currentEnrollment: 51,
+          honors: false,
+        }),
+        new Section({
+          ...dummySectionArgs,
+          sectionNum: '506',
+          grades: null,
+          instructor: new Instructor({ name: 'Tyagi' }),
+          currentEnrollment: 51,
+          honors: true,
+        }),
+      ];
+
+      const testSectionsSelected: SectionSelected[] = testSections.map((value) => ({
+        section: value,
+        meetings: [new Meeting({
+          ...dummyMeetingArgs,
+          section: value,
+        })],
+        selected: false,
+      }));
+
+      test('default sorting', async () => {
+        // arrange
+        const store = createStore(autoSchedulerReducer, applyMiddleware(thunk));
+        store.dispatch<any>(updateCourseCard(0, {
+          sections: [...testSectionsSelected],
+          customizationLevel: CustomizationLevel.SECTION,
+        }, '201931'));
+
+        // act
+        await store.dispatch<any>(updateSortType(0, SortType.DEFAULT));
+
+        // assert
+        const { sections } = store.getState().courseCards[0];
+        expect(sections[0].section.sectionNum).toBe('501');
```

Comment:
It'll probably be easier to store the section number order as an array (e.g. `['501, '502', '503', ...]`), then iterate over both of them and assert that they match up, rather than having to do 6 lines for each test.

Reply:
done